ime/app/src/test/java/com/anysoftkeyboard/ime/AnySoftKeyboardKeyboardSubtypeTest.java

```diff
@@ -79,7 +79,8 @@ public void testAvailableSubtypesReported() {
                     1618259652,
                     -517805346,
                     -1601329810,
-                    -1835196376
+                    -1835196376,
```

Comment:
Why did you make this change?

Reply:
Sorry, sloppy workspace management :grimacing: . I'm adding a new layout on another branch and got things mixed up.

I've removed that stuff and done a rebase and squash on the latest master to tidy it all up.